docs/src/python/user-guide/expressions/user-defined-functions.py

```diff
@@ -11,22 +11,25 @@
         "values": [10, 7, 1],
     }
 )
+print(df)
+# --8<-- [end:dataframe]
 
+# --8<-- [start:shift_map_batches]
 out = df.group_by("keys", maintain_order=True).agg(
-    pl.col("values").map_batches(lambda s: s.shift()).alias("shift_map"),
+    pl.col("values").map_batches(lambda s: s.shift()).alias("shift_map_batches"),
     pl.col("values").shift().alias("shift_expression"),
 )
-print(df)
-# --8<-- [end:dataframe]
+print(out)
+# --8<-- [end:shift_map_batches]
```

Comment:
currently, this snippet creates `df`, then creates `out`, then prints `df`. But `out` is never used - instead, in the `.md` file, the output of `out` is hard-coded.

I'm suggesting to, instead, split the snippet into two:
- create `df`, and show it
- create `out`, and show it, without hard-coding any output

Reply:
Sounds good.